spec/features/studios/show_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "Studio Index Page" do
+  before(:each) do
+    test_data
+  end
+
+  describe '#Viewing the Studio Show Page - US 4' do
+    it 'when viewing the show page, there is the studio name and location' do
+      visit "/studios/#{@studioghibli.id}"
+      expect(page).to have_content(@studioghibli.name)
+      expect(page).to have_content(@studioghibli.location)
+    end
+
+    it 'also includes a unique list of all the actors that have been in any of the studio movies' do
+      visit "/studios/#{@studioghibli.id}"
+      studio_actors = []
+      @studioghibli.movies.each{|movie| studio_actors << movie.actors}
```

Comment:
Be careful about condensing code like this in a test - it can seem slick at the time, but can become harder to read or to refactor later. Sometimes it is better in the long run to write it all out.